etc/avocado/avocado.conf

```diff
@@ -38,6 +38,14 @@ utf8 =
 # Keep job temporary files after jobs (useful for avocado debugging)
 keep_tmp_files = False
 
+[remoter.behavior]
+# Reject unknown SSH host keys.
+# 'False' will leave you wide open to man-in-the-middle attacks!
+# 'True' will only work with RSA keys (due to a bug in Paramiko).
+# If using 'True', accept the remote host key fingerprint by using:
+#   $ ssh -oHostKeyAlgorithms='ssh-rsa' <host>
+reject_unknown_hosts = False
```

Comment:
Do you think it'd be useful to also add `env.disable_known_hosts` to allow testing in offline-testing-farms? @adereis, @clebergnu